db/migrate/20170224171300_create_order_items.rb

```diff
@@ -0,0 +1,11 @@
+class CreateOrderItems < ActiveRecord::Migration[5.0]
+  def change
+    create_table :order_items do |t|
+      t.references :order, foreign_key: true
+      t.references :item, foreign_key: true
+      t.integer :quantity
+
+      t.timestamps
+    end
+  end
+end
```

Comment:
I think we could go either way. I think it would be easier to just make a method in the model. It doesn't seem like something we need just yet.